include/maxscale/session.h

```diff
@@ -481,4 +486,24 @@ MXS_SESSION* session_get_current();
  **/
 uint64_t session_get_current_id();
 
+/**
+ * @brief DCB callback for upstream throtting
+ *
```

Comment:
Adding some more comments on when these are called and for what reason would help to explain their intended usage.

For example, explaining why and when the `session_upstream_throttle_callback` function is called and what happens when throttling is enabled or disabled.